main.go

```diff
@@ -416,8 +416,8 @@ func installOption(listAll bool, custBinPath, mirrorURL *string) {
 	tflist = lib.RemoveDuplicateVersions(tflist)
 
 	templates := promptui.SelectTemplates{
-		Active:   `»{{ .Version | green | bold }}`,
-		Inactive: `{{ .Version | cyan }}`,
+		Active:   `{{ "»" | green | bold }} {{ .Version | green | bold }}`,
```

Comment:
I feel like `»` being outside parenthesis was on purpose: to visually separate currently selected version and other versions available though not yet selected.

Reply:
With "outside" I meant that this char doesn't get styled with color and bold font.
To be honest I don't really get which selection menu (?) this is about and hence can't see it myself =(

Reply:
Is that built from my branch? It should look something like this.
<img width="281" alt="Screen Shot 2022-05-16 at 8 15 35 AM" src="https://user-images.githubusercontent.com/10012479/168613591-08f94a8b-ac06-4070-b925-dbee7f7c1b24.png">

Reply:
So before this commit `»` wasn't styled in the prompt so this brings it in line with the output of the versioning.

Reply:
Oh, my. I should have better not get myself in front of my laptop today at all 🤦🏻 Please disregard 😺 Going to get myself some more coffee probably 🤣